solution/graph_traversal/1743/main.py

```diff
@@ -0,0 +1,40 @@
+# Authored by : seonjoo0885
+# Co-authored by : -
+# Link : http://boj.kr/1b8f001e0d3241ada8e1074f213f88d4
+
+import sys
+from collections import deque
+def input():
+    return sys.stdin.readline().rstrip()
+
+def bfs(x, y):
+    q = deque([(x, y)])
+    visited[x][y] = 1
+    cnt = 1
+    while q:
+        x, y = q.popleft()
+        for i in range(4):
+            nx = x + dx[i]
+            ny = y + dy[i]
```

Comment:
python에서 list iteration을 할때에는 가능하면 인덱스로 순회하지 않고, 값으로 순회하는 것이 권장됩니다.
```suggestion
        for cx, cy in zip(dx, dy):
            nx = x + cx
            ny = y + cy
```

or
```suggestion
        for dx, dy in [(-1, 0), (1, 0), (0, -1), (0, 1)]:
            nx = x + dx
            ny = y + dy
```


조금 더 나은 방법으론 인접칸을 찾는 로직을 분리하는 것입니다.
```
def neighbors(x, y):
    for dx, dy in [(-1, 0), (1,0),(0,-1),(0,1)]:
        yield x+dx, y+dy
 
for nx, ny in neighbors(x, y):
     # blahblah
```

or

```
def candidates(x, y):
    for dx, dy in [(-1, 0), (1,0),(0,-1),(0,1)]:
        nx, ny = x+dx, y+dy
        if 0 <= nx < n and 0 <= ny < m and graph[nx][ny] == '#' and not visited[nx][ny]:
            yield nx, ny
 
for nx, ny in candidates(x, y):
     # blahblah
```